RPIPs/RPIP-34.md

```diff
@@ -0,0 +1,365 @@
+* Any requirements of this nature SHOULD be detailed under a specific heading in the same or another RPIP. 
+  * This heading should then be referenced in the VOP definition under 'Other Requirements'.
+
+---
+
+### VOP Usage - Instances
+> VOP-GMC-MEMBERSELECT-1 is a single GMC membership selection, with a date, candidates, a vote, and results.
+
+* A VOP instance is an instantiation of a VOP definition.  
+* When a VOP instance is triggered, it SHALL be assigned the next free ID number by the responsible party.
+  * ID numbers are unique per VOP definition, start at 1, and increment by 1. 
+  * VOP IDs MUST be appended to the VOP code with a hyphen.
+
+---
+
+### VOP Usage - Records
+> The outcome of the VOP-GMC-MEMBERSELECT-1 instance is recorded in the appropriate record VOP record section.
+
+* A VOP record entry contains the results of a concluded VOP instance.
+* A VOP record section MUST be included in the RPIP in which a VOP is defined.
+  * The VOP record section SHOULD follow the template defined under the VOP Record Section Template heading of this specification.
+  
+* When a VOP instance concludes, its outcome MUST be recorded in the appropriate VOP record section by the RPIP Editors. 
+  * A VOP Record MUST include the date the VOP instance concluded.
+  * A VOP Record MUST include an appropriate URL link indicating the outcome of the VOP.
+  * A VOP Record MUST include a brief, human-readable summary of the outcome.
+
+---
+
+### VOP Usage - Overview
+> A Living informational RPIP that lists the currently defined VOPs and the RPIPs in which they are defined.
+
+* The VOP overview is a summary of defined VOPs within Rocket Pool.  
+* The VOP overview SHOULD exist as a living, informational RPIP maintained by the RPIP Editors.
+* Whenever an RPIP is ratified, the RPIP Editors MUST update the VOP Overview RPIP to include an entry for any newly ratified VOPs.
+  * The VOP overview entry SHOULD follow the template defined under the VOP Overview Entry Template heading of this specification.
+
+---
+
+### VOP Common Requirements
+This section lists requirements and common rules that apply to all usage of the VOP framework.
+
+* The result of any VOP instance SHALL NOT be considered valid until it is made publicly accessible via the communication mediums commonly used by the Rocket Pool DAOs.
+
+---
+
+### VOP Templates and Examples
+This section provides templates and examples for effective interaction with the VOP framework.
+
+
+#### VOP Definition Template
+```
+### <VOP CODE>
+This is a voted operational process (VOP) definition, see [RPIP-34](RPIP-34.md) for more information.
+
+* **Name:** 
+* **Description:** 
+* **Purpose:** 
+* **Responsible Party:** 
+* **Voting Group:** 
+* **Vote Type:** 
+* **Trigger(s):** 
+  * 
+  * 
+  * 
+* **Other Requirements:**   
+  * 
+  *
+  *
+```
+
+##### VOP-EXAMPLE-EXAMPLE
```

Comment:
I would suggest using `VOP-GMC-MEMBERSELECT-1` as your ongoing concrete example, or using an abstract example above. I think the mix is not great. Could do Example Management Committee (EMC) if desired as an in betweener.

Reply:
Agree. Implemented in working copy.